### PR TITLE
Rework service UI

### PR DESCRIPTION
**Pull request by aboch**
Signed-off-by: Alessandro Boch <aboch@docker.com>

### CODE REVIEW

File: endpoint_info.go [hunk 1]
```diff
@@ -39,6 +39,12 @@ type InterfaceInfo interface {
 	AddressIPv6() net.IPNet
 }
 
+// ContainerInfo provides an interface to retrieve the info about the container attached to the endpoint
+type ContainerInfo interface {
+	ID() string
```
Comment (mrjana) [inline endpoint_info.go]: Can you please add comments to the methods?

File: client/service.go [hunk 2]
```diff
@@ -153,151 +169,175 @@ func (cli *NetworkCli) CmdServiceLs(chain string, args ...string) error {
...
-		fmt.Fprintln(wr, "NETWORK SERVICE ID\tNAME\tNETWORK")
+		fmt.Fprintln(wr, "SERVICE ID\tNAME\tNETWORK\tPROVIDER")
 	}
 
-	for _, networkResource := range endpointResources {
-		ID := networkResource.ID
-		netName := networkResource.Name
+	for _, sr := range serviceResources {
+		ID := sr.ID
+		bkID, err := getBackendID(cli, ID)
```
Comment (mrjana) [inline client/service.go]: Can you provide a way to truncated way of showing backend id?

File: endpoint_info.go (hunk 1 above)
Comment (aboch) [inline endpoint_info.go]: OK

File: client/service.go (hunk 2 above)
Comment (aboch) [inline client/service.go]: Thanks, I had it in my list, simply forgot.
Will push the change soon.

**Comment by aboch** [comment]
@mrjana I took care of your comments. Please take a look when you get a chance.

**Comment by mrjana** [comment]
LGTM

**Comment by mavenugo** [comment]
@aboch Thanks for handling this. LGTM.

**Pull request merged**